managing-os/hvm.md

```diff
@@ -295,11 +200,82 @@ HVM domains (including Windows VMs) can be [assigned PCI devices](/doc/assigning
 
 One problem at the moment however, is that after the whole system gets suspended into S3 sleep and subsequently resumed, some attached devices may stop working and should be restarted within the VM. This can be achieved under a Windows HVM by opening the Device Manager, selecting the actual device (such as a USB controller), 'Disabling' the device, and then 'Enabling' the device again. This is illustrated on the screenshot below:
 
-[![r2b1-win7-usb-disable.png](/attachment/wiki/HvmCreate/r2b1-win7-usb-disable.png)](/attachment/wiki/HvmCreate/r2b1-win7-usb-disable.png)
+[![r2b1-my-new-vm-usb-disable.png](/attachment/wiki/HvmCreate/r2b1-my-new-vm-usb-disable.png)](/attachment/wiki/HvmCreate/r2b1-my-new-vm-usb-disable.png)
+
+Converting VirtualBox VM to HVM
+-------------------------------
+
+Microsoft provides [free 90 day evaluation VirtualBox VMs for browser testing](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/).
+
+About 60 GB of disk space is required for conversion, use external harddrive if needed. Final root.img size is 40 GB.
+
+In Debian AppVM, install qemu-utils and unzip:
+
+~~~
+sudo apt install qemu-utils unzip
+~~~
+
+Unzip VirtualBox zip file:
+
+~~~
+unzip *.zip
+~~~
+
+Extract OVA tar archive:
+
+~~~
+tar -xvf *.ova
+~~~
+
+Convert vmdk to raw:
+
+~~~
+qemu-img convert -O raw *.vmdk win10.raw
+~~~
+
+Create new HVM in Dom0, with amount of RAM in MB you wish:
+
+~~~
+qvm-create --hvm win10 --label red --mem=4096
+~~~
+
+Copy file to Dom0:
+
+~~~
+qvm-run --pass-io untrusted 'cat "/media/user/externalhd/win10.raw"' > /var/lib/qubes/appvms/win10/root.img
```

Comment:
This will work only for R3.2 - on R4.0 by default LVM is used. On the other hand, if you copy that image to some temporary place (`/home/user` ?), then use `qvm-create ... --root-move-from=...`, then it will work on both versions.

Reply:
I see; they were not part of my instructions, I simply moved them to the end of the page without even looking at them. I fixed the instructions with your suggestion.